bentoml/server/api_server.py

```diff
@@ -183,14 +181,18 @@ def __init__(
 
         self.setup_routes()
 
-    def start(self):
+    def start(self, port: int, host: str = "127.0.0.1"):
```

Comment:
Can host be in the config?

Reply:
Yeah. But it should be done in another PR.